sklearn/utils/tests/test_extmath.py

```diff
@@ -568,6 +567,33 @@ def max_loading_is_positive(u, v):
     assert not v_based
 
 
+@pytest.mark.parametrize("n", [50, 100, 300])
+@pytest.mark.parametrize("m", [50, 100, 300])
+@pytest.mark.parametrize("k", [10, 20, 50])
+@pytest.mark.parametrize("seed", range(5))
+def test_randomized_svd_lapack_driver(n, m, k, seed):
+    # Check that different SVD drivers provide consistent results
+
+    # Matrix being compressed
+    rng = np.random.RandomState(seed)
+    X = rng.rand(n, m)
+
+    # Number of components
+    k = 10
```

Comment:
I think this originates from the mistake I made when suggesting and this has to be removed.
```suggestion
```